app/src/main/java/fr/free/nrw/commons/upload/UploadService.java

```diff
@@ -283,7 +283,13 @@ private void uploadContribution(Contribution contribution) {
                         String canonicalFilename = "File:" + uploadResult.getFilename();
                         Timber.d("Contribution upload success. Initiating Wikidata edit for entity id %s",
                                 contribution.getWikiDataEntityId());
+                        if (contribution.getDepictionsEntityIds() != null) {
+                            for (String s : contribution.getDepictionsEntityIds()) {
+                                wikidataEditService.createClaimWithLogging(s, canonicalFilename);
```

Comment:
Can we add a one liner comment of what it actually does

Reply:
added